package.json

```diff
@@ -17,14 +17,15 @@
 		"@ethersproject/units": "^5.6.1",
 		"@ethersproject/wallet": "^5.6.2",
 		"@reach/router": "^1.3.4",
+		"@swarm-city/ui-library": "^0.0.2",
 		"classnames": "^2.3.1",
 		"ethers": "^5.6.9",
 		"eventemitter3": "^4.0.7",
 		"preact": "^10.10.0",
 		"qrcode.react": "^3.1.0",
-		"react-easy-crop": "^4.4.0",
+		"react-easy-crop": "^4.4.1",
 		"teaful": "^0.10.0",
-		"wagmi": "^0.5.12"
+		"wagmi": "^0.6.0"
```

Comment:
Can we not update deps in this PR? I think that would make it much simpler/smaller/faster/better/easier.